src/components/Dashboard/FilingHistoryList/filings/DissolutionVoluntary.vue

```diff
@@ -145,6 +153,16 @@ export default class DissolutionVoluntary extends Vue {
   get actTitle (): string {
     return this.getDissolutionConfirmationResource?.act || '[unknown]'
   }
+  /** The court order file number. */
```

Comment:
Comment should be on line 157. I also noticed these two methods are duplicated countless times. Wonder if we move them to one of the stores. (not necessarily in this PR or ticket).

Reply:
They're specific to the current filing. Might make sense to have them in a "filings" mixin... in the future. (There are bigger things to fix.)

Reply:
> Comment should be on line 157.

Fixed.

Reply:
Deal, thanks. Resolving.